src/set_polkit_default_privs

```diff
@@ -44,50 +44,50 @@ esac
 # we need to use a useful order
 #
 # 1. common privileges file (we have none yet)
-files=""
+files=()
 
 # 2. easy, standard, restrictive as those are defined by SUSE
-have_local=
 for level in $POLKIT_DEFAULT_PRIVS; do
     case "$level" in
-	easy|standard|restrictive)
-	    if [ -e /etc/polkit-default-privs.$level ]; then
-		files="$files /etc/polkit-default-privs.$level"
-	    fi
-	;;
-	*)
-	    echo "invalid value '$level' in POLKIT_DEFAULT_PRIVS in /etc/sysconfig/security, ignoring" 1>&2
-	;;
+        easy|standard|restrictive)
+            if [ -e /usr/share/polkit-default-privs/profiles/$level ]; then
+                files+=("/usr/share/polkit-default-privs/profiles/$level")
```

Comment:
it could make sense to define this prefix `/usr/share/polkit-default-privs` in a common variable